src/components/Summary/TransactionalFolioNumber.vue

```diff
@@ -38,24 +38,26 @@
 </template>
 
 <script lang="ts">
-import { Component, Mixins, Emit, Vue, Watch, Prop } from 'vue-property-decorator'
+import { Component, Mixins, Prop, Vue, Watch } from 'vue-property-decorator'
 import { Action, Getter } from 'vuex-class'
 import { CommonMixin } from '@/mixins'
-import { ActionBindingIF, ValidFlagsIF } from '@/interfaces'
-// Shared Interfaces
-import { ContactPointIF } from '@bcrs-shared-components/interfaces'
+import { ActionBindingIF, FlagsReviewCertifyIF, FormFieldType } from '@/interfaces'
 
 @Component({})
 export default class TransactionalFolioNumber extends Mixins(CommonMixin) {
+  // Add element type to refs
+  $refs!: {
+    folioNumberInput: FormFieldType
+  }
+
   // Global getters
-  @Getter getUserEmail!: string
-  @Getter getBusinessContact!: ContactPointIF
-  @Getter getDocumentOptionalEmail!: string
-  @Getter getAlterationValidFlags!: ValidFlagsIF
+  @Getter getFolioNumber!: string
```

Comment:
what does the `!` do here?

Reply:
Yeah, it's a peculiarity of using the vuex store (which is both a static and runtime thing).

If you know C, you can think of this as an "extern" :)

Actually, I think with our current linter flags, omitting this doesn't cause a warning/error, but in the future it might.